OutOfSchool/OutOfSchool.WebApi/Controllers/V1/ApplicationController.cs

```diff
@@ -409,6 +470,24 @@ public async Task<IActionResult> AllowedToReview(Guid parentId, Guid workshopId)
         return Ok(await applicationService.AllowedToReview(parentId, workshopId).ConfigureAwait(false));
     }
 
+    /// <summary>
+    /// Method, that checks applications of SearchResult for 204 status code.
+    /// </summary>
+    /// <param name="applications">Applications of SearchResult</param>
+    /// <returns>
+    /// True - applications is null or applications is empty.
+    /// False - applications is not null and applications collection has minimum 1 application.
+    /// </returns>
+    private static bool CheckApplicationsFor204(SearchResult<ApplicationDto> applications)
```

Comment:
Now method appears as extension method for SearchResult.

Also, i really like your idea, that we can extract construction of "return NoContent(); return Ok();", now this construction is in ControllerExtensions. I saw, that a lot of controllers have this construction too, if you want, I can change this construction in another controllers as another task. I think, it is a good idea.

Reply:
P.S.

Method is extension for SearchResult, because, if it was like usual method, it will throw an ArgumentNullException, if instance of SearchResult is null.

Reply:
Thanks for refactoring.

> ... if you want, I can change this construction in another controllers as another task

That's good that you don't increase scope of current task. Would be good to create separate ticket.

Reply:
After we will finish this task, you can write this ticket and assign it to me.